pkg/sso/client.go

```diff
@@ -111,6 +112,9 @@ type GetAuthorizationURLOptions struct {
 	// The unique identifier for a WorkOS Connection.
 	Connection string
 
+	//  The unique identifier for a WorkOS Organization.
```

Comment:
Looks like there's an extra leading space here:

```suggestion
	// The unique identifier for a WorkOS Organization.
```